SCAutolib/controller.py

```diff
@@ -123,6 +131,16 @@ def setup_system(self, install_missing: bool, gdm: bool):
             logger.critical(msg)
             raise exceptions.SCAutolibException(msg)
 
+        if graphical:
+            run(['dnf', 'groupinstall', 'Server with GUI', '-y'])
```

Comment:
Just to make sure, will this keep the `Server with GUI` string intact? Because it may split the command into `dnf groupisntall Server with GUI -y` and that's not desired beahviour

Reply:
The string is kept intact; the command works as expected.